manifest.json

```diff
@@ -3,68 +3,142 @@
     {
       "path": "./index.md",
       "children": [
-        { "path": "about.md" },
-        { "path": "comparison.md" },
-        { "path": "feedback.md" }
+        {
```

Comment:
formatting changes look good to me, but we should check with @khorne3 since she has to look at this stuff more than I do :joy:

Reply:
:D